setup.cfg

```diff
@@ -16,3 +16,6 @@ ignore-files=(plot_.*.py|conf\.py)
 
 [wheel]
 universal=1
+
+[flake8]
+ignore=E402
```

Comment:
Maybe add a comment to explain which error it is and why it needs to be ignored in the context of flake8_diff.